climakitae/indices.py

```diff
@@ -0,0 +1,93 @@
+"""Functions for deriving fire indices"""
+
+import xarray as xr
+
+
+def fosberg_fire_index(t2_F, rh_percent, windspeed_mph):
+    """Compute the Fosberg Fire Weather Index.
+    Use hourly weather as inputs.
+    Ensure that the input variables are in the correct units (see below).
+
+    Parameters
+    ----------
+    t2_F: xr.DataArray
+        Air temperature in units of Fahrenheit
+    rh_percent: xr.DataArray
+        Relative humidity in units of 0-100 (percent)
+    windspeed_mph: xr.DataArray
+        Windspeed in units of miles per hour
+
+    Returns
+    -------
+    xr.DataArray
+        Fosberg Fire Weather Index computed for each grid cell
+
+    References
+    ----------
+    https://a.atmos.washington.edu/wrfrt/descript/definitions/fosbergindex.html
+    https://github.com/sharppy/SHARPpy/blob/main/sharppy/sharptab/fire.py
+    https://www.spc.noaa.gov/exper/firecomp/INFO/fosbinfo.html
+
+    """
+    # Compute the equilibrium moisture constant
+    m_low, m_mid, m_high = _equilibirum_moisture_constant(h=rh_percent, T=t2_F)
+
+    # For RH > 10%, use the low m value.
+    # For RH >= 10%, use the mid value
+    m = xr.where(rh_percent < 10, m_low, m_mid)
+    # For RH > 50%, use the high m value.
+    m = xr.where(rh_percent > 50, m_high, m)
```

Comment:
Just checking that there's not a missing line of code for the low m value here?

Reply:
The `low_m` is kind of sneakily in there: 
```
m = xr.where(rh_percent < 10, m_low, m_mid)
```
Where RH < 10%, use m_low. Where it's > 10% use m_mid.